.github/workflows/playground.yml

```
@@ -0,0 +1,44 @@
+name: Build Playground
+
+on:
+  push:
+    branches: [playground]
```

```suggestion
    branches: [main]
```
commit this before merging